src/main/java/org/broadinstitute/dsm/db/dao/settings/InstanceSettingsDao.java

```diff
@@ -92,6 +104,50 @@ public int delete(int id) {
         }
         return Optional.ofNullable((Boolean) results.resultValue);
     }
+  
+    public Optional<Boolean> getHasAddressTabByStudyInstanceName(String instanceName) {
```

Comment:
I think making this method to return just `boolean` type value would be better, wrapping Boolean into Optional looks like just extra work, which is not needed I think,  `getBoolean` method of `ResultSet` in any case returns either true or false in any case, even if result is `SQL NULL`, as the result, returning `result.resultValue` directly from the method should be fine.
Why I think that wrapping Boolean into Optional does not worth is that, we just have either `true` or `false`, manipulating them using Optional does not give any benefit, it could even make things a little bit complicated on later use of method which returns `Optional<Boolean>`.
If I miss something, please correct.